language/vm/vm-runtime/src/interpreter.rs

```diff
@@ -62,7 +62,7 @@ lazy_static! {
 
 // Names for special functions and structs
 lazy_static! {
-    static ref CREATE_ACCOUNT_NAME: Identifier = Identifier::new("make").unwrap();
+    static ref CREATE_ACCOUNT_NAME: Identifier = Identifier::new("create_account").unwrap();
```

Comment:
I also see a `CREATE_ACCOUNT_NAME` in `txn_executor.rs` that needs to be updated. In fact, several of these magic names are duplicated there. Should we unify these lists?

Reply:
I totally agree that we should unify these list of special names. I'll address this issue during the MoveVM refactor.